src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -247,18 +270,7 @@ def initialize_model_from_path(
     if "sequence_length" in model_kwargs:
         model.seqlen = model_kwargs["sequence_length"]
 
-    teacher = (
-        AutoModelForCausalLM.from_pretrained(
-            model_args.distill_teacher,
-            **teacher_kwargs,
-        )
-        if model_args.distill_teacher is not None
-        else None
-    )
-    if teacher is not None and "sequence_length" in teacher_kwargs:
-        teacher.seqlen = teacher_kwargs["sequence_length"]
-
-    return teacher, model_path, model
+    return model, teacher
```

Comment:
by changing the signature of this function, should we be concerned about breaking changes for any users?

Reply:
Good question, we never use the model_path and theres only one location using it in text_geneation.py so should be ok!